README.md

```diff
@@ -11,11 +11,12 @@ name, separated by spaces.
 name.
 - Function must copy the whole content to new file.
 Example:
+
 ```python
 copy_file("cp file.txt file.txt")  # Does nothing
 
 copy_file("cp file.txt new_file.txt")
-open("file.txt").read() == open("new_file.txt").read()  # True
+open("app/file.txt").read() == open("new_file.txt").read()  # True
```

Comment:
Do not change readme

Reply:
I didn't change that file))) I don't know, why does it added "app/" there..

Reply:
Should i change this line to "open("file.txt").read() == open("new_file.txt").read()  # True"?